ckanext/switzerland/dcat/profiles.py

```diff
@@ -611,3 +652,125 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):  # noqa
     def graph_from_catalog(self, catalog_dict, catalog_ref):
         g = self.g
         g.add((catalog_ref, RDF.type, DCAT.Catalog))
+
+
+class SwissSchemaOrgProfile(SchemaOrgProfile, MultiLangProfile):
+    def _basic_fields_graph(self, dataset_ref, dataset_dict):
+        items = [
+            ('identifier', SCHEMA.identifier, None, Literal),
+            ('version', SCHEMA.version, ['dcat_version'], Literal),
+            ('issued', SCHEMA.datePublished, ['metadata_created'], Literal),
+            ('modified', SCHEMA.dateModified, ['metadata_modified'], Literal),
+            ('author', SCHEMA.author, ['contact_name', 'maintainer'], Literal),
+            ('url', SCHEMA.sameAs, None, Literal),
+        ]
+        self._add_triples_from_dict(dataset_dict, dataset_ref, items)
+
+        items = [
+            ('title', SCHEMA.name, None, Literal),
+            ('notes', SCHEMA.description, None, Literal),
+        ]
+        self._add_multilang_triples_from_dict(dataset_dict, dataset_ref, items)
+
+    def _list_fields_graph(self, dataset_ref, dataset_dict):
+        items = [
+            ('language', SCHEMA.inLanguage, None, Literal),
+        ]
+        self._add_list_triples_from_dict(dataset_dict, dataset_ref, items)
+
+    def _publisher_graph(self, dataset_ref, dataset_dict):
+        if any([
+            self._get_dataset_value(dataset_dict, 'publisher_uri'),
+            self._get_dataset_value(dataset_dict, 'publisher_name'),
+            dataset_dict.get('organization'),
+        ]):
+
+            publisher_uri = publisher_uri_from_dataset_dict(dataset_dict)
+            if publisher_uri:
+                publisher_details = URIRef(publisher_uri)
+            else:
+                # No organization nor publisher_uri
+                publisher_details = BNode()
+
+            self.g.add((publisher_details, RDF.type, SCHEMA.Organization))
+            self.g.add((dataset_ref, SCHEMA.publisher, publisher_details))
+            self.g.add((dataset_ref, SCHEMA.sourceOrganization, publisher_details))  # noqa
+
+            publisher_name = self._get_dataset_value(
+                dataset_dict,
+                'publisher_name'
+            )
+            if not publisher_name and dataset_dict.get('organization'):
+                publisher_name = dataset_dict['organization']['title']
+                self._add_multilang_value(
+                    publisher_details,
+                    SCHEMA.name,
+                    multilang_values=publisher_name
+                )
+            else:
+                g.add((publisher_details, SCHEMA.name, Literal(publisher_name)))  # noqa
+
+            contact_point = BNode()
+            self.g.add((publisher_details, SCHEMA.contactPoint, contact_point))
+
+            self.g.add((contact_point, SCHEMA.contactType, Literal('customer service')))  # noqa
```

Comment:
is this intended to be 'customer service' ?

Reply:
Yes